MaaSSim/maassim.py

```diff
@@ -18,7 +18,8 @@
 
 from MaaSSim.traveller import PassengerAgent, travellerEvent
 from MaaSSim.driver import VehicleAgent
-from MaaSSim.decisions import f_dummy_repos, f_match, dummy_False
+from MaaSSim.decisions import f_dummy_repos, dummy_False
+from decisions import f_match
```

Comment:
why not `from MaaSSim.decisions`?

Reply:
Yes so this is what I meant. In this line `decisions` refers to the python file in the MaaSSim folder, `MaaSSim.decisions` refers to the file from the MaaSSim package. I can not change the package file, only the folder in MaaSSim.

Reply:
So I think we need to discuss how we handle the MaaSSim package. If you want to edit files in the package we currently run into issues because you need to change the locations from which you import.